src/native/src/3rdparty/json/ChangeLog.md

```diff
@@ -0,0 +1,224 @@
+# Change Log
```

Comment:
Just going to mention one more time that most of the dependencies will be downloaded by cmake and compiled at the first run if needed. The ones that are in-tree now are meant to live in your source tree and be compiled with your project.